openedx_events/apps.py

```diff
@@ -20,6 +23,13 @@ def general_signal_handler(sender, signal, **kwargs):  # pylint: disable=unused-
     #        "topic_a": { "event_key_field": "my.key.field", "enabled": True },
     #        "topic_b": { "event_key_field": "my.key.field", "enabled": False }
     # }"
+    if kwargs.get(SIGNAL_PROCESSED_FROM_EVENT_BUS) is True:
+        logger.debug(
+            "Declining to send signal to the Event Bus since that's"
```

Comment:
[nit] Missing space (same in unit test)
```suggestion
            "Declining to send signal to the Event Bus since that's "
```